xds/internal/balancer/edsbalancer/eds_test.go

```diff
@@ -131,6 +132,10 @@ func (f *fakeEDSBalancer) handleEDSResponse(edsResp xdsclient.EndpointsUpdate) {
 
 func (f *fakeEDSBalancer) updateState(priority priorityType, s balancer.State) {}
 
+func (f *fakeEDSBalancer) updateServiceRequestsCounter(serviceName string) {
+	f.serviceName.Send(serviceName)
```

Comment:
> Where should that be verified? I'm guessing added to one of the waitFor functions below?

Yes. And call it to verify in the test.

Reply:
Made this change. Wasn't 100% sure what test you are talking about, added the check to `TestConfigChildPolicyUpdate` because it is the only test that invokes the policy update and has a `fakeEDSBalancer`.